cmd/bundle/plan.go

```diff
@@ -28,10 +28,12 @@ func newPlanCommand() *cobra.Command {
 
 	var force bool
 	var clusterId string
+	var output string
```

Comment:
The output flag is configured by the root command.

You can get its value with `root.OutputType(cmd)`.

For example:
```go
switch root.OutputType(cmd) {
case flags.OutputText:
	// ...
case flags.OutputJSON:
	// ...
}
```